VL.ImGui.Stride/src/ImGuiRenderer.cs

```diff
@@ -0,0 +1,205 @@
+﻿using ImGuiNET;
+using Stride.Input;
+using Stride.Graphics;
+using Stride.Rendering;
+using Stride.Core.Mathematics;
+using System.Runtime.CompilerServices;
+using Buffer = Stride.Graphics.Buffer;
+using RendererBase = Stride.Rendering.RendererBase;
+
+using VL.Core;
+using VL.Lib.Collections;
+using VL.Stride;
+using VL.Stride.Rendering;
+using VL.Stride.Input;
+
+using System.Reactive.Disposables;
+
+namespace VL.ImGui
+{
+    using ImGui = ImGuiNET.ImGui;
+
+    public partial class ImGuiRenderer : RendererBase, IDisposable
+    {
+        const int INITIAL_VERTEX_BUFFER_SIZE = 128;
+        const int INITIAL_INDEX_BUFFER_SIZE = 128;
+
+        // dependencies
+        GraphicsDevice device;
```

Comment:
Should be `IResourceHandle<GraphicsDevice>` deviceHandle, and `device => deviceHandle.Resource`. Or of course safe it in a separate field. For details see comment below in ctor.